pkg/networkservice/common/mechanisms/sendfd/client.go

```diff
@@ -68,7 +68,7 @@ func (s *sendFDClient) Request(ctx context.Context, request *networkservice.Netw
 
 func (s *sendFDClient) Close(ctx context.Context, conn *networkservice.Connection, opts ...grpc.CallOption) (*empty.Empty, error) {
 	rpcCredentials := grpcfd.PerRPCCredentials(grpcfd.PerRPCCredentialsFromCallOptions(opts...))
-	opts = append(opts, grpc.PerRPCCredentials(rpcCredentials))
+	opts = append([]grpc.CallOption{grpc.PerRPCCredentials(rpcCredentials)}, opts...)
```

Comment:
Why do we need to change the order of call options?

Reply:
My fault, I did a mistake during the testing, there is no need for this.